lib/cfg-grammar.y

```diff
@@ -1104,16 +1104,16 @@ driver_option
     ;
 
 threaded_dest_driver_option
-	: KW_RETRIES '(' LL_NUMBER ')'
+	: KW_RETRIES '(' positive_integer ')'
         {
           log_threaded_dest_driver_set_max_retries(last_driver, $3);
         }
 
 dest_driver_option
         /* NOTE: plugins need to set "last_driver" in order to incorporate this rule in their grammar */
 
-	: KW_LOG_FIFO_SIZE '(' LL_NUMBER ')'	{ ((LogDestDriver *) last_driver)->log_fifo_size = $3; }
-	| KW_THROTTLE '(' LL_NUMBER ')'         { ((LogDestDriver *) last_driver)->throttle = $3; }
+	: KW_LOG_FIFO_SIZE '(' nonnegative_integer ')'	{ ((LogDestDriver *) last_driver)->log_fifo_size = $3; }
```

Comment:
This should be `positive_integer`.
Specifying this option to 0, all of your messages will be dropped:

```
 Destination queue full, dropping messages; queue_len='0'
```

:-1: